spack/packages/dla-future/package.py

```diff
@@ -26,7 +26,7 @@ class DlaFuture(CMakePackage, CudaPackage):
     depends_on("blaspp")
     depends_on("lapackpp")
     depends_on("hpx cxxstd=14 networking=none")
-    depends_on("hpx@1.5.0", when="~cuda")
+    depends_on("hpx@1.5.0:1.5.99", when="~cuda")
```

Comment:
Wasn't `hpx@1.5` equivalent to `hpx@1.5.<any patch version>`? Not that we're going to have more than 100 patch versions for 1.5, or more than 1 for that matter :D

Reply:
yeah...you're right! I always struggle with this...thanks for pointing it out!